webserver/webserver.go

```diff
@@ -18,13 +18,15 @@ var staticFiles embed.FS
 //go:embed templates/status.html
 var statusHTML string
 
+var log hclog.Logger
```

Comment:
Not a big deal, but instead of every package having to define it's own logger, you can create it once in local "log" package and import that everywhere else and use the single instance.

Reply:
in this PR we have removed webserver for now. this line is here just to stop package erroring. in next PR we need to modify webserver package to work with new crd.